tests/ActionBuilder/Cart/SetBillingAddressTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace BestIt\CommercetoolsODM\Tests\ActionBuilder\Cart;
+
+use BestIt\CommercetoolsODM\ActionBuilder\Cart\SetBillingAddress;
+use BestIt\CommercetoolsODM\Mapping\ClassMetadataInterface;
+use Commercetools\Core\Model\Cart\Cart;
+use Commercetools\Core\Request\Carts\Command\CartSetBillingAddressAction;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for the set billing address builder.
+ *
+ * @author Tim Kellner <tim.kellner@bestit-online.de>
+ * @package BestIt\CommercetoolsODM\Tests\ActionBuilder\Cart
+ */
+class SetBillingAddressTest extends TestCase
+{
```

Comment:
Maybe i forgot it last time ... BestIt\CommercetoolsODM\Tests\ActionBuilder\SupportTestTrait.